### ISSUE TITLE
some pulls/issues are not migrated on resume

### ISSUE THREAD

**Issue by ViliusS** [piceaTech/node-gitlab-2-github#167]
In some cases pull requests or issues are skipped. Such cases include:
* If multiple GitLab merge requests have the same words (like Jira ticket numbers) in title, and you stop and resume migration second time.
* If GitLab merge requests have exactly the same title, and you stop and resume migration second time.
* If somebody already started to work on GitHub repository and created a merge request or an issue with the same or similar name, and you try to migrate GitLab merge requests at later time.

Basically it skips at [this line](https://github.com/piceaTech/node-gitlab-2-github/blob/7e056e231d8599ba35d10824c3a59e0eaba9713c/src/index.ts#L528), however I'm struggling to think of a good way to solve this.
